src/main/java/org/threadly/examples/features/PrioritySchedulerExample.java

```diff
@@ -0,0 +1,164 @@
+package org.threadly.examples.features;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import org.threadly.concurrent.PriorityScheduler;
+import org.threadly.concurrent.TaskPriority;
+import org.threadly.concurrent.future.FutureUtils;
+import org.threadly.concurrent.future.ListenableFuture;
+
+/**
+ * Provides a basic example of the PriorityScheduler feature.
+ * 
+ * @author sreid8 - Sean Reid
+ *
+ */
+public class PrioritySchedulerExample {
+  
+  private static Boolean threadDetail = false;
+  
+  /**
+   * Main entry point of the example.
+   * @param args -> program args. run w/o args for help
+   */
+  public static void main(String[] args) {
```

Comment:
Does it make sense to have these feature examples able to be ran?  I had not considered it, but thinking about it now, I am thinking maybe not?  These don't seem interesting to run, and the code around execution just distracts from what we are really trying to demonstrate.

Let me know your thoughts, but my vote is to axe the main function and such.  For actual development, feel free to add /src/test/java/... and put unit tests in which can start the execution.  Don't worry about coverage, I am just mentioning that unit test could be a way to test execution of the feature example without a main function.  If you don't feel the unit tests are necessary, feel free to not include any (for this example work).